docker/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker run -p ${DOCKER_OCR_PORT:-8000}:8000 --mount type=bind,source="$(pwd)",target=/wikimedia-ocr wikimedia-ocr:latest
```

Comment:
This could have `source="$(cd "$(dirname $0)/.."; pwd)"` to guard against when it's run from a different directory.

Reply:
Thanks